torchcfm/conditional_flow_matching.py

```diff
@@ -152,10 +152,7 @@ def compute_conditional_flow(self, x0, x1, t, xt):
         del t, xt
         return x1 - x0
 
-    def sample_noise_like(self, x):
-        return torch.randn_like(x)
-
-    def sample_location_and_conditional_flow(self, x0, x1, t=None, return_noise=False):
+    def sample_location_and_conditional_flow(self, x0, x1, t=None, return_noise=False, eps=None):
```

Comment:
can you run some notebooks to ensure the behaviour is still correct please? thx.